### ISSUE TITLE
Shell cannot work well on iOS and macOS

### ISSUE THREAD

**Issue by shugaoye** [dotnet/maui#5599]
### Description

I created a shell app using Xamarin.Forms template and port it to .NET MAUI. It work well on Windows and Android, but the UI looks not correct on iOS and maccatalyst.

### Steps to Reproduce

1. Create a new Xamarin.Forms app using the Visual Studio template and select Shell
2. Port it to .NET MAUI
3. Test it on Windows, Android, iOS and maccatalyst

It works well on Windows and Android.
On iOS and maccatalyst, only menu is shown, but the content page on the right doesn't show.

How can I submit the screenshots?

### Version with bug

Preview 14 (current)

### Last version that worked well

Unknown/Other

### Affected platforms

iOS, macOS

### Affected platform versions

iOS 15.2, macOS 11.3

**Comment by shugaoye** [comment]
Screenshot is here:
![error image](https://github.com/shugaoye/shugaoye.github.io/blob/maui/images/maui_ios_5599.png?raw=true)

**Comment by DavidBerschauer** [comment]
Same here. Did you solve it?

**Comment by codemonkey85** [comment]
I'm still having this issue - even with the default app template. Shell works in Windows, not in iOS / iPadOS / macOS. If in `App.xaml.cs` I set `MainPage = new AppShell()`, I get what you see in the second screenshot (macOS). If I set `MainPage = new MainPage()`, the MainPage content is shown correctly (but then I have no shell).

![2022-04-22](https://user-images.githubusercontent.com/1414036/164723374-71952d48-3671-4708-89e8-a5b55329ebb4.png)
![Screen Shot 2022-04-22 at 8 54 28 AM](https://user-images.githubusercontent.com/1414036/164723403-9d0b94ba-7432-40f9-9318-4e88787ae0cc.png)
